### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AJV009**
A minor spell check!
prequesities -> prerequisite

There isn't any word such "prequesities" on google. Looks like a non-dictionary word 

Supporting Documents:
https://dictionary.cambridge.org/dictionary/english/prerequisite
https://www.merriam-webster.com/dictionary/prerequisite

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ To build the documentation:
 
 1. Build and install PyTorch
 
-2. Install the prequesities
+2. Install the prerequisite
```
Comment (zou3519) [inline CONTRIBUTING.md:241]: nit: "Install the prerequisites"
(plural because there are multiple prerequisites)

**Review by zou3519 — COMMENTED** [review]
I'm happy to land this as-is but since we're fixing this line anyways we should improve the grammar

File: CONTRIBUTING.md (hunk 1 above)
Comment (AJV009) [reply CONTRIBUTING.md:241]: Am sorry I missed the plural thing!

**Comment by AJV009** [comment]
Am not a hard core coder to understand the underlying PyTorch code but want to contribute to great open source projects in some way so should I go over every md files and do some grammar fixes? 😁😁😁

File: CONTRIBUTING.md (hunk 1 above)
Comment (zou3519) [reply CONTRIBUTING.md:241]: Oh no, don't apologize - thank you for finding the typo and fixing it!

**Comment by zou3519** [comment]
@AJV009 we always appreciate typo fixes because a lot of the devs don't go through the the documentation unless we're writing new documentation :). Feel free to submit more PRs and tag me as a reviewer

**Comment by AJV009** [comment]
> 
> 
> @AJV009 we always appreciate typo fixes because a lot of the devs don't go through the the documentation unless we're writing new documentation :). Feel free to submit more PRs and tag me as a reviewer

Sure, I'll make another branch with as many fixes I find and submit a PR!

**Comment by AJV009** [comment]
> 
> 
> @AJV009 we always appreciate typo fixes because a lot of the devs don't go through the the documentation unless we're writing new documentation :). Feel free to submit more PRs and tag me as a reviewer

Another pull request -> https://github.com/pytorch/pytorch/pull/41599
Am sorry for the mess in making commits and PR's because am kind of new to all this!

**Comment by AJV009** [comment]
Already merged in another PR.
Good to close!

**Pull request closed without merging**